application/apps/indexer/processor/src/search.rs

```diff
@@ -149,29 +157,34 @@ impl SearchHolder {
                 .join("|")
         );
         let matcher = RegexMatcher::new(&regex)?;
+        let mut matchers: Vec<Regex> = vec![];
+        for filter in self.search_filters.iter() {
+            if let Ok(r) = Regex::from_str(&filter.value.clone()) {
```

Comment:
missing error handling. maybe we can even to this when creating the search holder and making it fail when a regex cannot be built.